src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/vedtak/Vedtaksstatistikk.kt

```diff
@@ -10,68 +10,88 @@ import no.nav.tilleggsstonader.sak.utbetaling.tilkjentytelse.domain.AndelTilkjen
 import no.nav.tilleggsstonader.sak.vilkår.stønadsperiode.dto.StønadsperiodeDto
 import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.Vilkårsresultat
 import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.DelvilkårDto
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.VilkårDto
 import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.regler.RegelId
 import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.AktivitetType
 import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.MålgruppeType
 import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.ResultatVilkårperiode
 import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.Vilkårperiode
 import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.VilkårperiodeType
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Column
 import java.time.LocalDate
 import java.time.LocalDateTime
 import java.util.UUID
 
-// TODO: Vurder om dette bør flyttes til kontrakter
-
-class VedtaksstatistikkDvh(
+data class Vedtaksstatistikk(
+    @Id
+    val id: UUID = UUID.randomUUID(),
     val fagsakId: UUID,
     val behandlingId: UUID,
+    val eksternFagsakId: Long,
     val eksternBehandlingId: Long,
     val relatertBehandlingId: Long?, // Ekstern behandlingsid på relatert behandling
     val adressebeskyttelse: AdressebeskyttelseDvh,
     val tidspunktVedtak: LocalDateTime,
-    val målgruppe: List<MålgruppeDvh>,
-    val aktivitet: List<AktivitetDvh>,
-    val vilkårsvurderinger: List<VilkårsvurderingDvh>,
+    @Column("målgruppe")
```

Comment:
vi pleier å unngå æøå i databasen tror jeg. Burde denne hete "malgruppe"

Reply:
Burde vell også være i flertall siden det er en liste

Reply:
https://nav-it.slack.com/archives/CGRMQHT50/p1715071547778399?thread_ts=1715069509.838689&cid=CGRMQHT50